R-package/DESCRIPTION

```diff
@@ -4,7 +4,8 @@ Title: Light Gradient Boosting Machine
 Version: ~~VERSION~~
 Date: ~~DATE~~
 Authors@R: c(
-    person("Guolin", "Ke", email = "guolin.ke@microsoft.com", role = c("aut", "cre")),
+    person("Yu", "Shi", email = "", role = c("aut", "cre")),
```

Comment:
@shiyu1994 

1. Are you comfortable being the maintainer for the R package, for CRAN purposes? I'm proposing that because I assume Microsoft requires that it be a Microsoft employee.
2. Can you please provide an email address you'd like to use here? This is required.

Reply:
Sure. Please use this email address: yushi2@microsoft.com. Thanks!

Reply:
```suggestion
    person("Yu", "Shi", email = "yushi2@microsoft.com", role = c("aut", "cre")),
```